cli/src/main.rs

```diff
@@ -51,6 +51,12 @@ struct Args {
     /// Only output the transaction calldata without any extra information.
     #[arg(short, long)]
     quiet: bool,
+
+    /// Quiet mode.
+    ///
+    /// Only output the transaction calldata without any extra information.
+    #[arg(short, long)]
+    inputs: bool,
```

Comment:
I liked your suggestion of calling this `params` to print out the `createProxyWithNonce` parameters.

The issue with it not working with the existing `prefix` flag is that `#[arg(short)]` defaults to using the first letter of there field name. You can fix that by specifying something else, for example:

```suggestion
    #[arg(short = 'P', long)]
    params: bool,
```

This way, `-P` is used to print parameters, and `-p` is used for setting a prefix.